ui/app/components/clients/charts/line.ts

```diff
@@ -9,8 +9,9 @@ import { parseAPITimestamp } from 'core/utils/date-formatters';
 import { format, isValid } from 'date-fns';
 import { debug } from '@ember/debug';
 
-import type { Count, MonthlyChartData, Timestamp } from 'vault/vault/charts/client-counts';
 import type ClientsVersionHistoryModel from 'vault/models/clients/version-history';
+import type { MonthlyChartData, Timestamp } from 'vault/vault/charts/client-counts';
+import type { TotalClients } from 'vault/vault/utils/client-count-utils';
```

Comment:
So client types only need to be added in one place in the future I reused the total client declaration for `Count`